ui/src/testing/stubs/cohort-review-service-stub.ts

```diff
@@ -106,10 +106,6 @@ const cohortChartDataStub = {
 };
 
 export class CohortReviewServiceStub extends CohortReviewApi {
-  configuration;
```

Comment:
What were these doing? They do not seem like function calls.

Reply:
I do not know. I focused exclusively on making the type checker happy.